app/src/renderer/vuex/modules/validators.js

```diff
@@ -54,6 +59,29 @@ export default ({ node }) => {
       if (validatorHash === state.validatorHash) return
       commit(`setValidatorHash`, validatorHash)
       await dispatch(`getValidators`)
+    },
+    async getValidatorDistributionInfoAndRewards({ commit }, validatorAddr) {
```

Comment:
can this be split into two functions? 

`getValidatorDistributionInfo()`
`getValidatorRewards()`

Reply:
I agree

Reply:
done